src/session/Cargo.toml

```diff
@@ -22,6 +22,5 @@ common-telemetry.workspace = true
 common-time.workspace = true
 derive_builder.workspace = true
 derive_more = { version = "1", default-features = false, features = ["debug"] }
-meter-core.workspace = true
```

Comment:
@shuiyisong Does 3rd party crates rely on this?

Reply:
I don't think so. We can remove this now.